iphone/Maps/Classes/FirstSession.mm

```diff
@@ -55,11 +55,14 @@ of this software and associated documentation files (the "Software"), to deal
 static NSString * const kFirstLaunchDateKey = @"AlohalyticsFirstLaunchDate";
 static NSString * const kTotalSecondsInTheApp = @"AlohalyticsTotalSecondsInTheApp";
 static NSString * const kIsAlohalyticsDisabledKey = @"AlohalyticsDisabledKey";
+static NSString * const kCFBundleShortVersionStringKey = @"CFBundleShortVersionString";
+static NSString * const kUDLastAppVersionPromptedForReview = @"kUDLastVersionPromptedForReview";
 
 // Used to calculate session length and total time spent in the app.
 // setup should be called to activate counting.
 static NSDate * gSessionStartTime = nil;
-static BOOL gIsFirstSession = NO;
+static BOOL gIsFirstSession = false;
```

Comment:
What's the motivation for this change?
For C++ true/false you need to use C++ `bool` type.

Reply:
I'll set both NO